database/index.js

```diff
@@ -38,22 +38,63 @@ const createWatchInfo = (watchId, watchName, options, callback) => {
 };
 
 const getWatchInfo = (watchId, callback) => {
-  const queryStringWatchInfo = `SELECT * FROM watches WHERE watches.wid = ${watchId};`;
-  const queryStringStrapInfo = `SELECT * FROM watches, strap_options, straps WHERE watches.id = strap_options.watch_id AND straps.id = strap_options.strap_id AND watches.wid = ${watchId};`;
+
+  // SELECT name, pathfilename
```

Comment:
I'm assuming that this commented out line and the ones following were previous attempts and notes. On the next refactor, just remember to remove them so your code looks cleaner!